home/utils/Environment.ts

```diff
@@ -13,8 +13,15 @@ const IsIOSRestrictedBuild =
   Platform.OS === 'ios' &&
   Kernel.iosClientReleaseType === Kernel.ExpoClientReleaseType.APPLE_APP_STORE;
 
+function isIOS14(): boolean {
+  if (Platform.OS !== 'ios') return false;
+  const versionNumber = parseInt(String(Platform.Version), 10);
+  return versionNumber >= 14;
+}
+
```

Comment:
I think it's far simpler just to check if we're on iOS. Then we don't have to answer support questions about why a feature disappeared when they upgraded their phone.

Reply:
it's disappearing regardless, at least it's clear that the change is related to iOS 14 pasteboard functionality rather than something we just removed.

Reply:
It just seems harder to maintain two code paths. Now our QA (on iOS 14 probably) isn't going to test the iOS 13 code path.

Reply:
fair enough